garage/logger/snapshotter.py

```diff
@@ -0,0 +1,88 @@
+"""Allows the taking of snapshots."""
+from os import path as osp
+import pickle
+
+
+class Snapshotter(object):
+    """This singleton handles the creation of snapshots."""
```

Comment:
https://en.wikipedia.org/wiki/Singleton_pattern

> ...a software design pattern that __restricts the instantiation of a class to one__.

This object can be instantiated as many times as I want.

Reply:
Fixed